index.js

```diff
@@ -20,7 +19,7 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/slack', slack.router);
-app.use('/channels', channels);
+app.use(channels);
```

Comment:
@kurtishouser I noticed you moved the `/channels` route into the router, but I think to stay consistent on this part of the codebase it would make more sense to revert this line to how it was. Thanks!